packages/contracts/contracts/abstracts/libs/logic/AtomicSwapStateLogic.sol

```diff
@@ -63,11 +63,11 @@ library AtomicSwapStateLogic {
             amount: bidMsg.bidAmount,
             order: bidMsg.orderID,
             status: IAtomicSwapBase.BidStatus.Placed,
-            bidder: bidMsg.bidder,
+            bidder: msg.sender,
             receiveTimestamp: block.timestamp,
             expireTimestamp: bidMsg.expireTimestamp
         });
-        bids[bidMsg.orderID][bidMsg.bidder] = newBid;
```

Comment:
to select bidder is provide opportunity for aggregators and other libraries. 
And then, it will be used for inter-chainswap later. so it's reasonable to use bidder parameter.